pages/index.tsx

```diff
@@ -296,7 +312,9 @@ let availableTokens = await staking.availableTokens();
 
             <div className="row pb-3">
               <div className="col-md-12">
-                <h4 className="h3 color-blue">Latest activity</h4>
+                <a target="_blank" href="https://github.com/bepronetwork">
+                  <h4 className="h3 color-blue">Latest activity</h4>
+                </a>
               </div>
             </div>
 
```

Comment:
Another issue we forgot to add was 

- Remove the prData array render and replace it with 3 columns linking to "bepro-js", "web-network" e "landing"